File: t1/SEP3BlazorT1Client/SEP3BlazorT1Client/Models/GuestReview.cs
using System;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;

namespace SEP3BlazorT1Client.Models
{
    public class GuestReview
    {
        [Required]
        [JsonProperty("rating")]
        public double Rating { get; set; }
        [JsonProperty("text")]
        public string Text { get; set; }
        
        [JsonProperty("createdDate")]
        public DateTime CreatedDate { get; set; }
        [Required]
        [JsonProperty("hostId")]
        public int HostId { get; set; }
        
        [Required]
        [JsonProperty("guestId")]
        public int GuestId { get; set; }
    }
}

File: t3/viabnb.tier3.dataserver.rest/src/main/java/dk/viabnb/sep3/group6/dataserver/rest/t3/models/GuestReview.java
package dk.viabnb.sep3.group6.dataserver.rest.t3.models;

import java.time.LocalDate;

public class GuestReview
{

  private double rating;
  private String text;
  private LocalDate createdDate;
  private int guestId;
  private int hostId;

  public GuestReview(double rating, String text,  LocalDate createdDate, int guestId, int hostId)
  {
    this.rating = rating;
    this.text = text;
    this.createdDate = createdDate;
    this.guestId = guestId;
    this.hostId=hostId;
  }

  public double getRating()
  {
    return rating;
  }

  public void setRating(double rating)
  {
    this.rating = rating;
  }

  public String getText()
  {
    return text;
  }

  public void setText(String text)
  {
    this.text = text;
  }

  public LocalDate getCreatedDate() {
    return createdDate;
  }

  public void setCreatedDate(LocalDate createdDate) {
    this.createdDate = createdDate;
  }

  public int getGuestId() {
    return guestId;
  }

  public int getHostId() {
    return hostId;
  }

  public void setGuestId(int guestId) {
    this.guestId = guestId;
  }

  public void setHostId(int hostId) {
    this.hostId = hostId;
  }
}
